src/main/java/frc/robot/Robot.java

```diff
@@ -22,11 +41,12 @@ public void disabledPeriodic() {
 
     @Override
     public void teleopInit() {
-
+        shooter.stop();
     }
 
     @Override
     public void teleopPeriodic() {
+        intake.print();
```

Comment:
move the ```shoter.print()``` to the ```teleopPeriodic()``` or the ```intake.print()``` to the ```robotPeriodic()```

Reply:
1. both should be in `robotPeriodic` to receive updates in all modes
2. `print` is probably the wrong name since we aren't "printing" to a stream, we are writing to smartdashboard/shuffleboard (networktables)
3. Instead of creating a method of your own, override `periodic` in the subsystems and put that logic there. It will be automatically called periodically as  with `robotPeriodic`

Reply:
I fixed it.